### PR TITLE
Refactor quote-summary-actions component

### PR DESCRIPTION
**Pull request by Larisa-Staroverova**

### CODE REVIEW

File: feature-libs/quote/components/summary/actions/quote-summary-actions.component.ts [hunk 1]
```diff
@@ -74,35 +72,7 @@ export class QuoteSummaryActionsComponent
...
-    [this.PADDING_BLOCK_START, '1.5rem'],
-    [this.PADDING_BLOCK_END, '1.5rem'],
-    [this.POSITION, 'fixed'],
-  ];
-
-  desktopStyling: readonly [property: string, value: string][] = [
-    [this.WIDTH, '100%'],
-    [this.PADDING_BLOCK_START, '1rem'],
-    [this.POSITION, 'static'],
-  ];
```
Comment (Larisa-Staroverova) [inline feature-libs/quote/components/summary/actions/quote-summary-actions.component.ts:105]: Hi @Platonn, 
as you suggested, I removed the styling manipulation completely.

File: feature-libs/quote/components/summary/actions/quote-summary-actions.component.ts [hunk 2]
```diff
@@ -138,10 +108,6 @@ export class QuoteSummaryActionsComponent
     });
   }
 
-  protected isMobile(): Observable<boolean> {
```
Comment (Larisa-Staroverova) [inline feature-libs/quote/components/summary/actions/quote-summary-actions.component.ts:141]: I got rid of `breakpointService` as well

File: feature-libs/quote/components/summary/actions/quote-summary-actions.component.ts [hunk 3]
```diff
@@ -173,141 +139,65 @@ export class QuoteSummaryActionsComponent
    * @protected
    */
   protected adjustBottomProperty(): void {
```
Comment (Larisa-Staroverova) [inline feature-libs/quote/components/summary/actions/quote-summary-actions.component.ts:141]: @Platonn,
one thing that could / should be done is to find out a way how to force the action buttons to stay before BottomHeaderSlot and not continue sliding above it.

File: feature-libs/quote/core/services/quote-storefront-utils.service.ts [hunk 4]
```diff
@@ -41,19 +41,6 @@ export class QuoteStorefrontUtilsService {
     }
   }
 
-  /**
-   * Removes styling for element.
-   *
-   * @param querySelector - query selector
-   * @param property - CSS property
-   */
-  removeStyling(querySelector: string, property: string): void {
```
Comment (Larisa-Staroverova) [inline feature-libs/quote/core/services/quote-storefront-utils.service.ts:50]: I got rid of this method completely.

**Comment by Platonn** [comment]
LGTM ✅

**Pull request merged**